Content/.template.config/template.json

```diff
@@ -81,6 +81,12 @@
       "dataType": "bool",
       "defaultValue": "false",
       "description": "use NPM instead of default Yarn for JS package management"
+    },
+    "Azure": {

```

Comment:
How about introducing a `choice` parameter, say "Deploy" with two options for now: `Docker` and `Azure` - both options do not have much sense if combined

Reply:
That makes sense to me as well.

Reply:
Agreed. I've made that change.